mlflow_export_import/model/list_registered_models.py

```diff
@@ -7,9 +7,13 @@
 import click
 from mlflow_export_import.common.http_client import MlflowHttpClient
 
-@click.command()
+
+@click.command("list-models")
 @click.option("--output-dir", help="Output directory.", default=".", type=str)
 def main(output_dir):  # pragma: no cover
+    """
+    Lists all registered models.
+    """
```

Comment:
Here's where I tell the `main` function that it's click command is "list-models". Click will handle this for us automatically if the function being decorated with `click.command()` was named `list_models`.

I also place the docstring on the function - click will automatically pass this to the CLI